Aquality.Selenium/src/Aquality.Selenium/Browsers/BrowserTabNavigation.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Aquality.Selenium.Core.Localization;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Aquality.Selenium.Browsers
+{
+    public class BrowserTabNavigation : IBrowserTabNavigation
+    {
+        private readonly RemoteWebDriver driver;
+
+        internal BrowserTabNavigation(RemoteWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private ILocalizedLogger Logger => AqualityServices.LocalizedLogger;
+
+        public void CloseTab()
+        {
+            Logger.Info("loc.browser.tab.close");
+            driver.Close();
+        }
+
+        public string GetTabName()
```

Comment:
That isn't a tab name, it is a tab handle. tab name could be returned by the `driver.Title` property

Reply:
you are right, fixed